tool/site/client-react/src/components/prescription-tracker/prescriptionItems.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react"
+import {Table} from "nhsuk-react-components"
+import {PrescriptionProps} from "./prescription"
+import {Task} from "fhir/r4"
+import {getDispenseStatusExtension} from "../../fhir/customExtensions"
+
+export interface PrescriptionItemProps {
+  identifier: string
+  dispenseStatus: string
+}
+
+export function createPrescriptionItemProps(task: Task): Array<PrescriptionItemProps> {
+  return task.input.map(input => {
+    return {
+      identifier: input.valueReference.identifier.value,
+      dispenseStatus: getDispenseStatusExtension()(input.extension).valueCoding.display
+    }
+  })
+}
+
+export const PrescriptionItems: React.FC<PrescriptionProps> = ({prescriptionItems}) => {
+  return (
+    <Table.Panel heading="Items">
+      <Table caption="Item summary">
+        <Table.Head>
+          <Table.Row>
+            <Table.Cell>Identifier</Table.Cell>
+            <Table.Cell>Status</Table.Cell>
+          </Table.Row>
+        </Table.Head>
+        <Table.Body>
+          {prescriptionItems.map((item, index) => <ItemRow key={index} {...item} />)}
+        </Table.Body>
+      </Table>
+    </Table.Panel>
+  )
+}
+
+const ItemRow: React.FC<PrescriptionItemProps> = ({
```

Comment:
could you rename to `PrescriptionItemRow`? at first glance, itemRow seems like a generic name